frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/health.ts

```diff
@@ -33,29 +33,36 @@ type KubevirtHealthResponse = {
 export const additionalResources = {
   vmis: {
     isList: true,
-    kind: VirtualMachineInstanceModel.kind,
-  },
-  pods: {
-    isList: true,
-    kind: PodModel.kind,
+    kind: referenceForExtensionModel({
```

Comment:
you can replace all of these with `kubevirtReferenceForModel`

e.g. `kubevirtReferenceForModel(VirtualMachineInstanceModel)`